content/docs/3.0/tasks/configure/security.md

```diff
@@ -15,17 +15,70 @@ This page discusses how to secure your TiKV deployment. Learn how to:
 
 ## Transport Layer Security (TLS)
 
-Transport Layer Security protects TiKV communications from tampering or inspection. This is useful in data centers where there may be untrustworthy (or unpriviledged) users or binaries, when operating over a WAN, or fulfilling compliance demands.
+Transport Layer Security is an standard protocol for protecting networking communications from tampering or inspection. TiKV uses OpenSSL, an industry standard, to implement it's TLS encryption.
 
-To use TLS encryption with TiKV, you need to prepare the certificates for the deployment.
+It's often necessary to use TLS in situations where TiKV is being deployed or accessed from outside of a secure virtual local area network (VLAN). This includes deployments which cross the WAN (the public internet), which are part of an untrusted data center network, or where other untrustworthy users or services are active.
 
-### Prepare Certificates
+Before you get started, review your infrastructure. Your organization may already use something like the [Kubernetes certificates API](https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/) to issue certificates. You will need a **Certificate Authority** (CA) certificate, the key for that CA certificate, as well as individual unique **certificates**, and **keys** for each TiKV or PD service, and one or many certificates for TiKV clients depending on your needs. If you have these, you can skip the optional section below.
+
```

Comment:
```suggestion
## Configure TLS for TiKV

```

Reply:
This seems like a duplicate of the above heading?

Reply:
Sorry I totally forgot about this PR. This heading is intended as a second level that includes the configuration steps. The one above - actually I suggest we remove it. I looked at other docs in the repo and they don't have a 2nd level heading at the beginning of the document.

Reply:
The optional section below has nothing to do with configuring TiKV though, it's just how to roll keys for any application.